src/app/screens/Register/i18n.ts

```diff
@@ -0,0 +1,31 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'Register', {
+  logoAlt: 'logo Wolox',
+  firstName: 'Nombre',
+  lastName: 'Apellido',
+  email: 'Correo',
+  password: 'Contraseña',
+  passwordConfirmation: 'Confirmación de Contraseña',
+  btnLogin: 'Iniciar Sesión',
+  btnRegister: 'Registrarme',
+  btnLanguage: 'Ingles',
+  errorEmail: 'Asegúrece de escribir un formato de correo valido',
```

Comment:
its `Asegúrese` also, why not more simple like `Formato de email inválido`. And move to `~constants/i18n.ts` where the more general wording is placed.
`i18next.addResources('es', 'FormValidations', {`